recipes/pytest-pyvista/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pytest-pyvista" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pytest_pyvista/pytest_pyvista-{{ version }}.tar.gz
+  sha256: 43e6f0a85b6ff23aff51b7f750528de0fdda6da2011a94b636f05d8498fe38a0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
```

Comment:
@tkoyama010 As you're using `flit` to build and package `pytest-pyvista`, I'm guessing that you're going to have to add it here as a dependency.

This should be enough as you're using `flit` instead of `setuptools` for the build system and backend. 

BTW I've never used `flit` myself and I've never seen a conda-forge recipe using `flit` before, but I'll see if I can find an example somewhere (we only ever use `setuptools`).

Reply:
@tkoyama010 Okay, so you do need to include `flit_core >=3.2,<4` here in the `host` section, along with `pip` and `python` 👍

Reply:
Given the other `flit` package examples I found ([here](https://github.com/search?q=org%3Aconda-forge+%2Fflit-core%2F+path%3A%2F%5Erecipe%5C%2F%2F&type=code)) there is no need to include `setuptools` in the `host` section.

Reply:
```suggestion
    - python >=3.8
    - flit-core >=3.2,<4
```